ci-operator/config/openshift/extended-platform-tests/openshift-extended-platform-tests-master.yaml

```diff
@@ -0,0 +1,38 @@
+base_images:
+  base:
+    name: "4.4"
+    namespace: ocp
+    tag: base
+binary_build_commands: make build WHAT=cmd/extended-platform-tests
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.13
+images:
+- dockerfile_path: Dockerfile.rhel
+  from: base
+  inputs:
+    root:
+      as:
+      - builder
+      paths: null
+  to: extended-platform-tests
+promotion:
+  name: "4.4"
+  namespace: ocp
```

Comment:
@stevekuznetsov this config will produce an image `extended-platform-tests` . We will add this image as in initcontainer to the cluster-launch-installer-e2e.yaml template, so that we can enable a new suite `e2e-extended-platform` in 
 all openshift repos, eventually.  What do I need to do to make this image available to all?  Will this config do it? I'm reading docs, but will ask you, too.

Reply:
the image has the binary `extended-platform-tests` similar to very similar to `openshift-tests`

Reply:
@sallyom yeah promoting the image will make it available for anyone whose `tag_specification` is `ocp` namespace, which is pretty much anyone who is interested in using these tests, I guess